content/learn/getting-started/node-es6.fr.md

```diff
@@ -0,0 +1,39 @@
+---
+title: ecmascript-2015-es6-and-beyond
+displayTitle: 'ECMAScript 2015 (ES6) et plus'
+category: learn
+---
+
+Node.js est construit avec des versions modernes de [V8](https://v8.dev/). En restant à jour avec les dernières versions de ce moteur, nous nous assurons que les nouvelles fonctionnalités de la [spécification JavaScript ECMA-262](http://www.ecma-international.org/publications/standards/Ecma-262.htm) sont apportées aux développeurs Node.js en temps voulu, ainsi que des améliorations continues de la performance et de la stabilité.
+
+Toutes les fonctionnalités ECMAScript 2015 (ES6) sont réparties en trois groupes pour les fonctionnalités **en cours d'exécution**, **établies** et **en cours de réalisation** :
+
+* Toutes les fonctionnalités **shipping**, que V8 considère comme stables, sont activées **par défaut sur Node.js** et ne nécessitent **PAS** d'indicateur d'exécution.
+* Les fonctionnalités **Staged**, qui sont des fonctionnalités presque achevées et qui ne sont pas considérées comme stables par l'équipe V8, nécessitent un indicateur d'exécution : `--harmony`.
+* Les fonctionnalités **en cours** peuvent être activées individuellement par leur drapeau d'harmonie respectif, bien que cela soit fortement déconseillé, sauf à des fins de test. Note : ces drapeaux sont exposés par V8 et seront potentiellement modifiés sans aucune notification de dépréciation.ce.
```

Comment:
We should update this at some point, I think it's fine to consider ES2015 implementation done in V8, they are now working on more recent versions of the spec.

Attention tu traduis les termes dans la première ligne, mais pas dans la liste:
```suggestion
Toutes les fonctionnalités ECMAScript 2015 (ES6) sont réparties en trois groupes pour les fonctionnalités **en cours d'exécution**, **établies** et **en cours de réalisation** :

* Toutes les fonctionnalités **en cours d'exécution** (_shipping_), que V8 considère comme stables, sont activées **par défaut sur Node.js** et ne nécessitent **PAS** d'indicateur d'exécution.
* Les fonctionnalités **établies** (_staged_), qui sont des fonctionnalités presque achevées et qui ne sont pas considérées comme stables par l'équipe V8, nécessitent un indicateur d'exécution : `--harmony`.
* Les fonctionnalités **en cours** (_in progress_) peuvent être activées individuellement par leur drapeau d'harmonie respectif, bien que cela soit fortement déconseillé, sauf à des fins de test. Note : ces drapeaux sont exposés par V8 et seront potentiellement modifiés sans aucune notification de dépréciation.
```